ui/platform_window/x11/x11_window_ozone.h

```diff
@@ -35,6 +35,17 @@ class X11_WINDOW_EXPORT X11WindowOzone : public X11WindowBase,
   bool DispatchXEvent(XEvent* event) override;
 
  private:
+  // Calls OnLostCapture() and xwindow().
+  friend X11WindowManagerOzone;
+
+  // Called by |window_manager_| once capture is set to another xwindow.
+  void OnLostCapture();
+
+  // Converts events' location if they are originally from background windows,
```

Comment:
add a TODO to share code with DesktopWindowTreeHostX11::ConvertEventToDifferentHost.